src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -234,20 +242,39 @@ def __init__(
             # FP16 smallest positive subnormal works well here
             self.alphas_cumprod[-1] = 2**-24
 
-        sigmas = (((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5).flip(0)
-        timesteps = np.linspace(0, num_train_timesteps - 1, num_train_timesteps, dtype=float)[::-1].copy()
+        if use_flow_match:
+            timestep_offset = 1
+        else:
+            timestep_offset = 0
+
+        timesteps = np.linspace(
+            0 + timestep_offset, num_train_timesteps - 1 + timestep_offset, num_train_timesteps, dtype=float
+        )[::-1].copy()
         timesteps = torch.from_numpy(timesteps).to(dtype=torch.float32)
 
+        if use_flow_match:
+            sigmas = timesteps / num_train_timesteps
+            if not use_dynamic_shifting:
+                # when use_dynamic_shifting is True, we apply the timestep shifting on the fly based on the image resolution
+                sigmas = shift * sigmas / (1 + (shift - 1) * sigmas)
+        else:
+            sigmas = (((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5).flip(0)
+
         # setable values
         self.num_inference_steps = None
 
         # TODO: Support the full EDM scalings for all prediction types and timestep types
         if timestep_type == "continuous" and prediction_type == "v_prediction":
```

Comment:
is `v_prediction` relevant to flow matching? can people configure `use_flow_match` + `v_prediction`? based on the code, it is possible. But does this make sense?